docs/using-fit-on-fhir.md

```diff
@@ -2,18 +2,54 @@
 
 Congratulations. You have deployed Fit on FHIR to your Azure subscription and are ready to start collecting Google Fit Data. The first thing to do is have users on-board by authorizing sharing fitness data.
 
-## User authorization endpoint
+## User Authorization Endpoint
 
 When users navigate to the authorize endpoint, they are redirected to a Google sign-in page where they can enter their credentials and opt-in to sharing data with your Fit on FHIR instance. After they complete the authorization process, Fit on FHIR will continuously import the fitness data they opted to share. By default, data imports occur once per hour.
 
-**Your users can authorize sharing Google Fit data by navigating to [*https://authorize-{YOUR_BASE_NAME}.azurewebsites.net/api/googlefit/authorize*](#)**
-
 Your authorization endpoint is hosted by the Authorization Function. The endpoint to begin the authorization flow can be determined by viewing the Function overview page, finding the Function base URL and appending **api/googlefit/authorize** to the base.
 
+## Authorization Function Response Data
+
+To allow for users to be redirected to a Google sign-in page, the Authorization function returns a JSON object that contains two values:
+
+1. **AuthUrl** - This is the URL that should be used to navigate the user to the sign-in page.
+1. **ExpiresAt** - This is a timestamp that indicates when authorized access will expire, for the user that is trying to complete the authorization process.
+
 ## Finding your Authorization Function Base URL
 
 In your resource group, find the Authorization Function. It will be named **authorize-{YOUR_BASE_NAME}**, where, YOUR_BASE_NAME is the basename parameter provided when the resources were deployed.
 ![Auth Function in Resource Group](../media/auth-function-resource-group.png)
 
 In the overview section of the Function copy the base URL. Append the base URL with **api/googlefit/authorize**.
 ![Auth Function Base URL](../media/auth-function-url.png)
+
+## Configuring Access to your Authorization Function
+
+Access to the Authorization function can be configured to either allow for anonymous login, or login with authentication.  This setting is configured by setting
+*AuthenticationConfiguration__IsAnonymousLoginEnabled* to true for anonymous logins, and false for login with authentication.  This can be found in can be found in Settings->Configuration.
+
+**Required Query Parameters**:
+
+Anonymous login requests require two query parameters, *external_id* and *external_system*.  The *external_id* and *external_system* are used to create a FHIR [Identifier](http://hl7.org/fhir/datatypes.html#Identifier) that is stored in the [Patient Resource](http://hl7.org/fhir/patient.html). This Identifier can be used to link the Patient Resource to a user (or patient) in a different system.
```

Comment:
"Anonymous login requests require two query parameters"

I find this to be confusing. It says here, that 2 are required when there are actually 3 required parameters. It might be better to create a matrix that lists the supported query parameters what they do and when they are required.